cdk/src/open-data-platform/data-plane/schema/schema.sql

```diff
@@ -173,92 +197,22 @@ EXECUTE PROCEDURE update_last_update_timestamp();
 --- Create pre-computed tables.
 --- These are required to ensure acceptable latency for the tileserver.
 
--- Demographic aggregation tables.
-
-CREATE TABLE IF NOT EXISTS state_demographics
-(
-    census_geo_id         varchar(255) NOT NULL,
-    name                  varchar(255) NOT NULL,
-    black_population      real,
-    white_population      real,
-    total_population      real,
-    under_five_population real,
-    poverty_population    real,
-    geom                  geometry(Geometry, 3857),
-    PRIMARY KEY (census_geo_id)
-);
-CREATE INDEX IF NOT EXISTS geom_index ON state_demographics USING GIST (geom);
-
-CREATE TABLE IF NOT EXISTS county_demographics
-(
-    census_geo_id         varchar(255) NOT NULL,
-    name                  varchar(255) NOT NULL,
-    black_population      real,
-    white_population      real,
-    total_population      real,
-    under_five_population real,
-    poverty_population    real,
-    geom                  geometry(Geometry, 3857),
-    PRIMARY KEY (census_geo_id)
-);
-CREATE INDEX IF NOT EXISTS geom_index ON county_demographics USING GIST (geom);
-
--- TODO(breuch): Add insert statement once the zipcode -> demographics
--- connection established.
-CREATE TABLE IF NOT EXISTS zipcode_demographics
```

Comment:
This would break the API endpoint checked in today. Can you update the zip_codes/{zip_code}/scorecard API to reference the aggregated table instead of zipcode_demos?

Reply:
Done